guide/14-deep-learning/doccano setup guide.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<blockquote>verification:</blockquote><p> </p><p>I can't see this image, but I can see other images, in reviewnb app.</p><p>Kindly, check if this image has been added using base64 format.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Unable to reproduce

Reply:
@priyankatuteja can you kindly look into it.

Reply:
![image](https://user-images.githubusercontent.com/39754605/77623613-65978e80-6f66-11ea-9ed1-6864dc42b90b.png)

@akhilnegi Looks like this image is not added in the base64 format.

Reply:
fixed